cfgov/core/scripts/missing_migrations.py

```diff
@@ -11,17 +11,11 @@
 from django.core.management import call_command
 
 
-def run():
-    major, minor = VERSION[0:2]
-    if 1 == major and 10 <= minor:
-        print(
-            'makemigrations --exit deprecated in Django 1.10: '
-            'https://docs.djangoproject.com/en/1.10/ref/django-admin/'
-            '#cmdoption-makemigrations--exit'
-        )
-        sys.exit(1)
+# Django 1.10+ includes makemigrations --check, making this unnecessary
+is_django_110_plus = (1 == VERSION[0]) and (10 <= VERSION[1])
```

Comment:
True, it would. Change is made. It's saddening (but realistic) to think we might not be fully onto 1.10 when we're testing against 2.0 😢